Resources/Prototypes/Diseases/infectious.yml

```diff
@@ -118,35 +118,90 @@
   id: AMIV
   name: AMIV
   cureResist: 0.10
+  stages:
+    - 0
+    - 120
+    - 780
   effects:
     - !type:DiseasePopUp
       probability: 0.015
       type: Pvs
       message: disease-beat-chest-compulsion
       visualType: Medium
+      stages:
+        - 0
+        - 1
+        - 2
```

Comment:
?

Reply:
Stages is an array, and since I wanted this effect to apply to all three stages, I passed them in this way. I've tested it and it seems to work fine. Should I split it up into three effects?

Reply:
> ? The test is failing as the integers don't match.
> 
> If the stages are supposed to just match the indices then that's pretty jank and should probably have some kind of identifier.

Yea I had them originally set to match the integers, but the code was using the indices. I can update the DiseaseSystem code to use the time ints instead of the indices.

Reply:
I had a chat with emo on discord and updated the test in https://github.com/space-wizards/space-station-14/pull/9639

The effects themselves should probably be commented or something as right now reading through them is kind of an eye strain.